code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -158,14 +164,13 @@
 
 		if(modeled_location.blocks_air)
 
-			if((modeled_location.heat_capacity>0) && (partial_heat_capacity>0))
+			if((modeled_location.heat_capacity > 0) && (partial_heat_capacity > 0))
 				var/delta_temperature = air.temperature - target_temperature
 
-				var/heat = thermal_conductivity*delta_temperature* \
-					(partial_heat_capacity*target_heat_capacity/(partial_heat_capacity+target_heat_capacity))
+				var/heat = thermal_conductivity * delta_temperature * (partial_heat_capacity * target_heat_capacity / (partial_heat_capacity + target_heat_capacity))
```

Comment:
normally multiline code is bad, in this case I'm less against it because this  line mirrors our other more arcane heat movement procs. Not too big a deal tho